apps/sensenet/src/components/AddButton.tsx

```diff
@@ -51,15 +51,15 @@ const useStyles = makeStyles((theme: Theme) => {
     },
     listDropdown: {
       padding: '10px 0 10px 10px',
-      width: '245px',
+      minWidth: '245px',
       maxHeight: '548px',
+      width: 'auto',
```

Comment:
auto is its default value, so it is unnecessary to set

Reply:
@taki9 without setting with auto, the dropdown has a defult with and the items are cut in half